api/envoy/api/v2/route/route.proto

```diff
@@ -109,6 +109,12 @@ message VirtualHost {
   // specific; see the :ref:`HTTP filter documentation <config_http_filters>`
   // for if and how it is utilized.
   map<string, google.protobuf.Struct> per_filter_config = 12;
+
+  // <config_http_filters_router_x-envoy-original-path>` header.
+  // Decides whether the :ref:`x-envoy-attempt-count
```

Comment:
How this play with `suppress_envoy_headers` in router.proto?

Reply:
I'm fine either way, perhaps document the behavior when both are true? @htuch any suggestion?